time time
uint8 seqNum
uint8 imuId
AccelGyro data
